ccp/src/main/java/com/hbb20/Country.java

```diff
@@ -840,6 +891,7 @@ static int getFlagResID(Country country) {
         countries.add(new Country("sc", "248", "Seychelles"));
         countries.add(new Country("sl", "232", "Sierra Leone"));
         countries.add(new Country("sg", "65", "Singapore"));
+        countries.add(new Country("sxm", "1721", "Sint Maarten"));
```

Comment:
We use 2 letter, alpha-2 name code for country. For Sint Maarten it's **sx**. https://en.wikipedia.org/wiki/ISO_3166-1